### PR TITLE
add sha256 hash to the lists

### LINKED ISSUE #10379: Add SHA-256 hashes to file lists in solc-bin
Proposed by @axic in https://github.com/ethereum/solidity/pull/10352#discussion_r527268367.

> Hmm, didn't we also had a sh256 hash next to this? That is way more useful for anyone downloading a static compiler from here and trying to verify it on CI (as that works with shasum -a 256 <file>).

After doing this the example in solc-bin docs should be updated to include such a hash too.

### PR DESCRIPTION
**Pull request by ssi91**
Fixes https://github.com/ethereum/solidity/issues/10379

Relevant chances in the docs can be found there https://github.com/ethereum/solidity/pull/10932

### CODE REVIEW

File: update [hunk 1]
```diff
@@ -139,6 +139,7 @@ async function makeEntry (dir, parsedFileName, oldList) {
     const fileContent = await readFile(absolutePath)
     build.keccak256 = '0x' + ethUtil.keccak(fileContent).toString('hex')
     console.log("Computing hashes of '" + pathRelativeToRoot + "'")
+    build.sha256 = '0x' + ethUtil.sha256(fileContent).toString('hex')
```
Comment (chriseth) [inline update:142]: Can you also modify line 137 accordingly, please?

Comment (ssi91) [reply update:142]: @chriseth done. Mark as resolved if it's well now :)

**Review by cameel — DISMISSED** [review]
@chriseth Looks like the lint/update PR check did not run on this PR. And it's not just that it's not shown in the PR - I can't see it on CircleCI either: https://app.circleci.com/pipelines/github/ethereum/solc-bin. I wonder why. It does run on my PRs (see for example #80). Maybe it's something about permissions?

The PR itself seems fine but I'm marking it as "changes requested" to prevent merging for now - I think that PR is very important in this case.

**Review by cameel — APPROVED** [review]

**Comment by cameel** [comment]
Looks like the check did not run because this is a PR from an external repo. @axic has made the check mandatory now so at least we'll be able to see that it's missing but for security reasons we don't want to allow external PRs to run it.

@ssi91 I have resubmitted your PR as #82 and I'm closing this one.

**Pull request closed without merging**